project-base/app/src/FrontendApi/Resolver/Products/ProductsQuery.php

```diff
@@ -160,6 +180,34 @@ public function productsQuery(Argument $argument)
             $argument,
         );
 
+        if ($argument['categorySlug']) {
+            $category = $this->categoryQuery->categoryOrSeoMixByUuidOrUrlSlugQuery($info, urlSlug: $argument['categorySlug']);
+
+            return $this->productsByCategoryOrReadyCategorySeoMixQuery(
+                $argument,
+                $category,
+            );
+        }
+
+        if ($argument['brandSlug']) {
+            /** @var \App\Model\Product\Brand\Brand $brand */
+            $brand = $this->brandQuery->brandByUuidOrUrlSlugQuery(urlSlug: $argument['brandSlug']);
+
+            return $this->productsByBrandQuery(
+                $argument,
+                $brand,
+            );
+        }
+
+        if ($argument['flagSlug']) {
+            $flag = $this->flagQuery->flagByUuidOrUrlSlugQuery(urlSlug: $argument['flagSlug']);
```

Comment:
```suggestion
            $flag = $this->flagQuery->flagByUuidOrUrlSlugQuery($argument['flagSlug']);
```

Reply:
first argument is uuid, not slug

Reply:
After discussing the changes, I removed the named parameter and included `null` for the UUID.